virtwho.py

```diff
@@ -25,20 +25,62 @@
 import time
 from multiprocessing import Event, Queue
 import json
+
 import atexit
 from Queue import Empty
 from httplib import BadStatusLine
 
 from daemon import daemon
-from virt import Virt, DomainListReport, HostGuestAssociationReport, ErrorReport
+from virt import Virt, DomainListReport, HostGuestAssociationReport, ErrorReport, Hypervisor
 from manager import Manager, ManagerError, ManagerFatalError
+from manager.managerprocess import ManagerProcess
+from manager.subscriptionmanager import SubscriptionManager
 from config import Config, ConfigManager, InvalidPasswordFormat
 from password import InvalidKeyFile
+from datetime import datetime, timedelta
+from heapq import *
 
 import log
 
 from optparse import OptionParser, OptionGroup, SUPPRESS_HELP
 
+# Default interval for jobs to be delayed
+DefaultJobInterval = 60
```

Comment:
Reports are still sent as soon as they are collected. This presently only affects the retrieval of the status of async jobs from candlepin. What would be more reasonable job timeout?